internal/cephfs/util.go

```diff
@@ -45,7 +45,9 @@ func execCommand(ctx context.Context, program string, args ...string) (stdout, s
 	cmd.Stdout = &stdoutBuf
 	cmd.Stderr = &stderrBuf
 
-	klog.V(4).Infof(util.Log(ctx, "cephfs: EXEC %s %s"), program, sanitizedArgs)
+	if klog.V(util.Debug).Enabled() {
+		klog.InfoDepth(1, util.Log(ctx, "cephfs: EXEC %s %s"), program, sanitizedArgs)
```

Comment:
You do not need to use `InfoDepth` here, as the current function is the one that needs to be logged.

If you create a `util.Debugf()` function, it can replace the `util.Log(..)` *and* `klog.V(4).Infof(..)`. That new `util.Debug()` would need to use `InfoDepth()` with a `depth` of 1, so that the calling function/file/line of `util.Debug()` gets logged.